<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>위에서보기</string>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="video_viewer_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="insideCar_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>video_viewer_label</zorder>
  <zorder>insideCar</zorder>
  <zorder>insideCar_4</zorder>
  <zorder>insideCar_5</zorder>
  <zorder>insideCar_6</zorder>
  <zorder>insideCar_7</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
